deploy/helm-chart/values.yaml

```diff
@@ -85,6 +85,14 @@ serviceAccount:
   ## Autogenerated if not set or set to "".
   # name: nginx-gateway
 
+  ## The name of the secret containing docker registry credentials.
+  ## Secret must exist in the same namespace as the helm release.
+  imagePullSecretName: ""
+
+  ## A list of secret names containing docker registry credentials.
+  ## Secrets must exist in the same namespace as the helm release.
+  imagePullSecretsNames: []
```

Comment:
Since the field itself in the ServiceAccount is just called `imagePullSecret`, I wonder if we omit `Name` and `Names` from these fields, for consistency.